.github/workflows/main.yml

```diff
@@ -41,21 +28,32 @@ jobs:
         run: |
           make nodeup examples test
 
-  verify:
-    strategy:
-      matrix:
-        os: [ubuntu-18.04]
-        go: [1.13]
-      fail-fast: true
+  build-macos-amd64:
+    runs-on: macos-10.15
+    steps:
+    - name: Set up go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.14
+        stable: true
+
+    - uses: actions/checkout@v2
+      with:
+        path: ${{ env.GOPATH }}/src/k8s.io/kops
 
-    runs-on: ${{ matrix.os }}
+    - name: make nodeup examples test
```

Comment:
Can we remove nodeup here also?  Otherwise it's going to confuse me every time :-)

Reply:
Good point. Done.